install/eventstreams/templates/03-es.yaml

```diff
@@ -17,7 +17,7 @@ spec:
   apicurioRegistry: {}
   license:
     accept: {{ license_accept }}
-    license: L-KCVZ-JL5CRM
+    license: L-AUKS-FKVXVL
     use: EventAutomationNonProduction
   requestIbmServices:
```

Comment:
`spec.requestIbmServices` is deprecated in Event Streams 11.6.0 - lines 22-24 should be removed from this spec